database/migrations/2024_11_19_130731_create_beatmap_versioning_tables.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
```

Comment:
I suppose this comes from an ide. Not harmful but also not necessary as those are already aliased to just `DB` and `Schema` respectively

Reply:
I'm not even sure whether this was phpstorm or artisan at this point, but will trim